subsys/sd/sd.c

```diff
@@ -298,3 +298,22 @@ bool sd_is_card_present(const struct device *sdhc_dev)
 	}
 	return sdhc_card_present(sdhc_dev) == 1;
 }
+
+/* Enables SD card interrupt */
+int sd_enable_interrupt(const struct device *sdhc_dev,
+	sdhc_interrupt_cb_t callback, void *user_data)
+{
+	if (!(sdhc_dev && callback)) {
+		return -EINVAL;
+	}
+	return sdhc_enable_interrupt(sdhc_dev, callback, user_data);
+}
+
+/* Disable SD card interrupt */
+int sd_disable_interrupt(const struct device *sdhc_dev)
```

Comment:
what is the point of having these wrapper functions, if the caller still needs to know the sdhc device

Reply:
There really isn't a good reason for them to exist, the more I consider it. I'll drop these from the PR.